README.md

```
@@ -87,7 +87,7 @@ node arb-bot <OPTIONS>
 The app requires these arguments (all arguments can be set in env variables alternatively, more details below):
 - `-k` or `--key`, Private key of wallet that performs the transactions, one of this or --mnemonic should be specified. Will override the 'BOT_WALLET_PRIVATEKEY' in env variables
 - `-m` or `--mnemonic`, Mnemonic phrase of wallet that performs the transactions, one of this or --key should be specified, requires `--wallet-count` and `--topup-amount`. Will override the 'MNEMONIC' in env variables
-- `-r` or `--rpc`, RPC URL(s) that will be provider for interacting with evm, use different providers if more than 1 is specified to prevent banning. Will override the 'RPC_URL' in env variables
+- `-r` or `--rpc`, List of RPC url(s) for interacting with evm, optionally with selection weight and track size seperated by comma in form of key=value, example: url=https://rpc1.com,weight=0.5,trackSize=100 . Will override the 'RPC_URL' in env variables
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)